### ISSUE TITLE
/monitoring doesn't work on Google App Engine

### ISSUE THREAD

**Issue by davidkiss** [javamelody/javamelody#794]
I can access /monitoring?part=springBeans in GAE, but get below error for /monitoring:
```
javax.management.RuntimeErrorException: java.lang.InternalError: errno: 38 error: sysinfo failed to get swap size
	at com.sun.jmx.interceptor.DefaultMBeanServerInterceptor.rethrow(DefaultMBeanServerInterceptor.java:841) ~[na:1.8.0_112-google-v7]
	at com.sun.jmx.interceptor.DefaultMBeanServerInterceptor.rethrowMaybeMBeanException(DefaultMBeanServerInterceptor.java:852) ~[na:1.8.0_112-google-v7]
	at com.sun.jmx.interceptor.DefaultMBeanServerInterceptor.getAttribute(DefaultMBeanServerInterceptor.java:651) ~[na:1.8.0_112-google-v7]
	at com.sun.jmx.mbeanserver.JmxMBeanServer.getAttribute(JmxMBeanServer.java:678) ~[na:1.8.0_112-google-v7]
	at net.bull.javamelody.internal.model.MBeansAccessor.getAttribute(MBeansAccessor.java:91) ~[javamelody-core-1.74.0.jar:1.74.0]
	at net.bull.javamelody.internal.model.MBeansAccessor.getLongFromOperatingSystem(MBeansAccessor.java:100) ~[javamelody-core-1.74.0.jar:1.74.0]
	at net.bull.javamelody.internal.model.MemoryInformations.<init>(MemoryInformations.java:79) ~[javamelody-core-1.74.0.jar:1.74.0]
	at net.bull.javamelody.internal.model.JavaInformations.<init>(JavaInformations.java:143) ~[javamelody-core-1.74.0.jar:1.74.0]
	at net.bull.javamelody.internal.web.MonitoringController.doActionIfNeededAndReport(MonitoringController.java:158) ~[javamelody-core-1.74.0.jar:1.74.0]
	...
```

**Comment by evernat** [comment]
There was also a problem with [unavailable fonts](https://issuetracker.google.com/issues/72216727).
And this is fixed by 17623f7.
You can download a snapshot build at:
https://javamelody.ci.cloudbees.com/job/javamelody/net.bull.javamelody$javamelody-core/

**Comment by evernat** [comment]
Thanks for the issue.
